core-common/src/main/java/org/zalando/nakadi/domain/NakadiRecord.java

```diff
@@ -78,4 +82,15 @@ public NakadiRecord setPartition(final String partition) {
         this.partition = partition;
```

Comment:
probably partition should be part of the metadata

Reply:
especially, that we take partition for producerrecord from metadata :)

Reply:
This is done.